docs/source/notebooks/finch-usage.ipynb

```diff
@@ -126,7 +126,7 @@
           "output_type": "stream",
           "text": [
             "Process status:  ProcessSucceeded\n",
-            "Link to process output:  http://localhost:5000/outputs/9d475210-017c-11ee-b431-54ef33fc01a5/frost_days_sres_a2_experiment_20460101_20650101.nc\n"
+            "Link to process output:  /e353412c-c695-11ee-a6f6-484d7ef8da38/frost_days_sres_a2_experiment_20460101_20650101.nc\n"
```

Comment:
This change breaks Jenkins.  Is it normal the front part `http://localhost:5000/outputs/` disappeared from `urls.output` ?
```
19:46:37  _________ finch-master/docs/source/notebooks/finch-usage.ipynb::Cell 3 _________
19:46:37  Notebook cell execution failed
19:46:37  Cell 3: Cell outputs differ
19:46:37  
19:46:37  Input:
19:46:37  print("Process status: ", resp.status)
19:46:37  urls = resp.get()
19:46:37  print("Link to process output: ", urls.output)
19:46:37  
19:46:37  Traceback:
19:46:37   mismatch 'stdout'
19:46:37  
19:46:37   assert reference_output == test_output failed:
19:46:37  
19:46:37    'Process stat...20650101.nc\n' == 'Process stat...20650101.nc\n'
19:46:37    Skipping 49 identical leading characters in diff, use -v to show
19:46:37    -  output:  https://wps_host/wpsoutputs/STATUS_FILE/frost_days_sres_a2_experiment_20460101_20650101.nc
19:46:37    ?           ---------------------------
19:46:37    +  output:  /STATUS_FILE/frost_days_sres_a2_experiment_20460101_20650101.nc
```

Reply:
That's strange. I don't think I explicitly changed anything that affect this.

Reply:
Library changed ("Updated a few version pins (clisops, pre-commit hooks)" in the description) changing the output format during a notebook refresh?

In this case, a url without the `http://hostname` in front seems like a bug for me.